libs/vm/include/vm/module/free_function_invoke.hpp

```diff
@@ -20,32 +20,34 @@
 namespace fetch {
 namespace vm {
 
-template <typename ReturnType, typename FreeFunction, typename... Ts>
+template <typename ReturnType, typename Callable, typename... Ts>
 struct FreeFunctionInvokerHelper
 {
-  static void Invoke(VM *vm, int sp_offset, TypeId return_type_id, FreeFunction f,
+  static void Invoke(VM *vm, int sp_offset, TypeId return_type_id, Callable const &callable,
```

Comment:
Same as above, `Callable` is not even used on its own in `Invoke`'s body yet the argument is mandated to be `const &`.

Reply:
Like I said above, I don't think it makes a difference. This code needs some love badly anyway. I have an idea how to rearchitecture it the `*_invoke.hpp` headers where we don't have to repeat everything five (now - four) times and we don't pass around so many arguments without using them (`sp_offset` in the `Invoker` classes gives me nightmares). Let's have this conversation after I cut the code down to size.